src/components/SignUpForm/SignUpForm.tsx

```diff
@@ -16,36 +26,37 @@ const SignUpForm = () => {
     getValues,
     handleSubmit,
     formState: { errors },
-  } = useForm<IRegister>();
+  } = useForm<IRegister>({
+    resolver: yupResolver(validationSchema),
+  });
 
   const onSubmit = () => {
-    const data = getValues();
-    dispatch(setUserEmail(data.email));
-    dispatch(setUserPassword(data.password));
-    localStorage.setItem("user", data.email);
+    const { email } = getValues();
+    dispatch(setUserEmail(email));
```

Comment:
why you keep email in different places? For what email in redux?

Reply:
we need it for account page, and for showing  the result of game. i can remove it from redux and take from local storage.